src/crimson/os/seastore/transaction_manager.cc

```diff
@@ -309,6 +309,39 @@ TransactionManager::submit_transaction_direct(
     trim_alloc_to);
 }
 
+TransactionManager::update_lba_mappings_ret
+TransactionManager::update_lba_mappings(
+  Transaction &t)
+{
+  LOG_PREFIX(TransactionManager::update_lba_mappings);
+  SUBTRACET(seastore_t, "update extent lba mappings", t);
+  return seastar::do_with(
+    std::list<LogicalCachedExtentRef>(),
+    std::list<CachedExtentRef>(),
+    [this, &t](auto &lextents, auto &pextents) {
+    t.for_each_fresh_block_pre_submit([&lextents, &pextents](auto &extent) {
+      if (extent->is_logical()) {
+        // for rewritten extents, last_committed_crc should have been set
```

Comment:
Is it better?
```
// for rewritten extents, last_committed_crc should have been set
// because the crc of the original extent may be reused.
// also see rewrite_logical_extent()
```

Reply:
Yeah, will do